packages/ember-truth-helpers/package.json

```diff
@@ -8,67 +8,72 @@
   "repository": "https://github.com/jmurphyau/ember-truth-helpers",
   "license": "MIT",
   "author": "James Murphy",
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "default": "./dist/index.js"
-    },
-    "./*": {
-      "types": "./dist/*.d.ts",
-      "default": "./dist/*.js"
-    },
-    "./addon-main.js": "./addon-main.js"
-  },
   "files": [
-    "addon-main.js",
+    "addon-main.cjs",
+    "declarations",
     "dist"
   ],
   "scripts": {
-    "build": "rollup --config",
-    "copyDoc": "cp ../../*.md .",
-    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
-    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "build": "concurrently 'npm:build:*'",
+    "build:js": "rollup --config",
+    "build:types": "glint --declaration && grep -r -l '///' declarations/ | sort | uniq | xargs perl -e \"s/\\/\\/\\/.*$//\" -pi",
```

Comment:
I couldn't figure out why tsc leaves references like

```
/// <reference types="ember-source/types/preview/@ember/component/helper" />
/// <reference types="@glint/environment-ember-loose/-private/dsl/integration-declarations" />
```

without removing it, test-app fails to build.

not perfect, leave as is till we figure out the root cause